These are notes and code inspired by the book: "Expressed Sequence Tags (ESTs)", edited by J. Parkinson, 2009, Humana Press and springer.com. 

ESTs are single-pass  short (usually <1000 bp) reads, derived fron cDNA clones; it is a highly cost-effective way to discover genes. It is valuable for genome annotation and as an alternative to microarray analysis, for finding relative levels of gene expression. NGS (in 2009) made it cost-effective for small labs also to undertake EST. 

Ch. 1: EST Overview: ESTs are fragments of mRNA sequences (often overlapping) that are expressed via combination with randomly selected clones from cDNA libraries. They represent expressed portion of a genome, from either a whole organism or specific tissues - hence very useful for novel gene discovery and verification of gene predictions. As of 2013 (not sure why no update since then), there were 74,186,692 entries in dbEST, of which 8,704,790 were human entries. dbEST accepts only single-pass reads, not assembled sequences. A noted inefficiency is the repeated sequencing of highly abundant mRNAs. So, methods exist to 'normalize' the relative abundance of each cDNA in the library. The db has been used to explore eukaryotic phylogeny. **Ch. 11 discusses AutoFACT pipeline for customizing a BLAST-based pipeline**. Functional annotation algorithms, perhaps based on HMM models, is feasible today (since AA/peptide annotations are available). In proteomics, peptide fragments are matched to known protein sequences using tandem mass spectrometry. But such known sequences are limited (as of 2009) and ESTs have been used to id and characterize alternative splicing events, difficult to predict from the raw genome sequence. **Ch. 15 discusses current applications of ESTs in human disease.** 

Ch. 11: EST Databases and Web Tools for EST Projects: Focus is on the design of an EST database for a specific EST project. Uses TBestDB (of protists; Taxonomically-Broad eukaryote EST DB). Uses PostgresSQL, Perl, PHP to build the db and interface; AutoFACT for data processing and annotation. .... P. 249: Identifying the protein sequence that corresponds to a consensus (or contig) EST sequence is challenging. Assigne function to an unknown seq by similarity search aainst seqs of known functions, as performed with BLAST. Some are 'hyopthetical protein' - not very useful. AutoFACT is particularly useful for investigating new taxa. Written in Perl for Linux. Input is nucleotide sequence and conducts BLAST  search against miulitple dbs. Error rate <2%. Other tools: SpliceNest visualizes gene structure and alternative splicing based on EST clusters. Others: GEPSIS and SimiTri. 

Ch. 15: ESTs in Human Health. Past 10 years- complete characterization of the human transcriptome, to (1) id of all human genes, (2) transcriptome variability,  (3) spatial and temporal expression patterns of transcripts and variants, and (4) the association of transcripts and variants with complex biological phenomena, such as diseases. **High throughput technologies (EST, SAGE, MPSS, and microarrays) have made this feasible.** Some tissues from mammalian species are underrepresented in GenBank as well as several pathological situations. **Craig Venter (1991)** used EST in their breakthrough process.  

From [wik](https://en.wikipedia.org/wiki/Craig_Venter): In 2014 Venter co-founded Human Longevity, Inc., a company focused on extending the healthy, "high performance" human lifespan. The company said that it plans to sequence 40,000 genomes per year, with an initial focus on cancer genomes and the genomes of cancer patients. Human Longevity's mission is to extend healthy human lifespan by the use of high-resolution big data diagnostics from genomics, metabolomics, microbiomics, and proteomics, and the use of stem cell therapy. See this [article](http://www.lifeextension.com/Magazine/2014/11/Massive-Database-Planned-To-Crack-The-Secrets-Of-Aging/Page-01) in LifeExtension magazine

Glossary of terms:
* EST - Expressed Sequence Tag
* NGS - Next Generation Sequencing
* cDNA - Complementary DNA
* [dbEST](https://www.ncbi.nlm.nih.gov/dbEST/) - public EST database resource
* SAGE - Serial Analysis of Gene Expression
* MPSS - Massively Parallel Signature Sequencing
